app/auth/user_management.py

```
@@ -0,0 +1,79 @@
+    try:
+        user_record = auth.get_user(current_user["id"])
+        custom_claims = auth.get_custom_user_claims(user_record.uid)
+        return {
+            "id": user_record.uid,
+            "email": user_record.email,
+            "first_name": custom_claims["first_name"],
+            "last_name": custom_claims["last_name"],
+            "is_active": not user_record.disabled,
+            "created_at": str(user_record.user_metadata.creation_timestamp)
+        }
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)